src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/ByteArrayArgumentBindingProvider.cs

```diff
@@ -1,83 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Reflection;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Host.Bindings;
-using Microsoft.Azure.WebJobs.Host.Storage.Queue;
-
-namespace Microsoft.Azure.WebJobs.Host.Queues.Bindings
-{
-    internal class ByteArrayArgumentBindingProvider : IQueueArgumentBindingProvider
-    {
-        private readonly IContextGetter<IMessageEnqueuedWatcher> _messageEnqueuedWatcherGetter;
-
-        public ByteArrayArgumentBindingProvider(IContextGetter<IMessageEnqueuedWatcher> messageEnqueuedWatcherGetter)
```

Comment:
This IMessageEnqueuedWatcher functionality looks like it is being removed with no replacement? Isn't that breaking things?